go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/golangci/golangci-lint v1.53.3
```

Comment:
It would be great to minimize the changes in go.mod for this PR. Consider splitting it up.